internal/db/start/start.go

```diff
@@ -56,7 +56,6 @@ func NewContainerConfig() container.Config {
 	env := []string{
 		"POSTGRES_PASSWORD=" + utils.Config.Db.Password,
 		"POSTGRES_HOST=/var/run/postgresql",
-		"POSTGRES_INITDB_ARGS=--lc-ctype=C.UTF-8",
```

Comment:
is this intentional? we need this to set the same locale as platform db

Reply:
Yeah this is override down the line by an if/else statement: https://github.com/supabase/cli/pull/2796/files/de420f3044ea89994162aea89953559299968ba5#diff-f635ef2693ae7184092f49b3ed3c642a7d3bf0e727de4ff03ea207591c1ef0c2R62-R73

So seemed to me that this doesn't apply.